src/services/__tests__/QueueService.int.spec.ts

```diff
@@ -73,9 +79,21 @@ describe("QueueService tests", () => {
         expect(resultJoin).toHaveLength(1);
         expect(receivedQueueTime).toEqual(expectedQueueTime);
       });
-    });
+      it("In Active Match | Does not add to queue", async () => {
+        const mockMatchId = faker.datatype.uuid();
+        const mockPlayers = ActiveMatchBuilder.many(6, { matchId: mockMatchId });
+        await manuallyAddActiveMatch(mockPlayers);
```

Comment:
Do you need to manually add an active match if you're mocking the response of the isPlayerInActiveMatch call anyways?

Reply:
No.  Removed.

Reply:
Fixed.